### DISCUSSION TITLE
32 Bit add please

### DISCUSSION THREAD

**Discussion by Quazer34** [Flam3rboy/discord-bot-client#216] [Ideas]
32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit add please 32 Bit pls

**Comment by samuelscheit** [comment]
stop spamming

**Comment by samuelscheit** [comment]
what os are you using?

**Comment by fourteentrees** [comment]
I don't think the libraries DBC uses are supported for 32 bit systems
